spec/services/nomis/elite2/offender_api_spec.rb

```diff
@@ -59,13 +59,13 @@
       expect(response).to eq('C')
     end
 
-    it 'returns null if unable to find prisoner', :raven_intercept_exception,
+    it 'returns if unable to find prisoner',
```

Comment:
Although this tests the OffenderApi, is it worth having a test for OffenderService.get_offender() as well?